macos/FluentUI/Button/Button.swift

```diff
@@ -154,6 +154,39 @@ open class Button: NSButton {
 		}
 	}
 
+	/// When set to a non `nil` value ,the Secondary Image will always be placed on the trailing edge of the button in both LTR and
+	/// RTL language settings. It is designed to remain independent of  the `imagePosition` value of the NSButton which deals
+	/// with the main Button `image`.
+	public var secondaryImage: NSImage? {
+			didSet {
+				guard oldValue != secondaryImage else {
+					return
+				}
+
+				guard let cell = cell as? ButtonCell else {
+					return
```

Comment:
This one is probably a preconditionFailure rather than a silent failure as it is probably illegal to use a Fluent button with a different cell?